iOS/Flows/Home/_New/new_ConversationViewController.swift

```diff
@@ -82,76 +99,56 @@ class new_ConversationViewController: FullScreenViewController, CollectionViewIn
             self.conversationController.deleteMessage(message.id)
         }
     }
-
-    override func viewDidLayoutSubviews() {
-        super.viewDidLayoutSubviews()
-
-        self.blurView.expandToSuperviewSize()
-        self.collectionView.expandToSuperviewSize()
-    }
 }
 
 extension new_ConversationViewController: UICollectionViewDelegate {
 
+
     func collectionView(_ collectionView: UICollectionView,
                         willDisplay cell: UICollectionViewCell,
                         forItemAt indexPath: IndexPath) {
 
         // If all the messages are loaded, no need to fetch more.
         guard !self.conversationController.hasLoadedAllPreviousMessages else { return }
 
-        // Only start fetching new messages once the user is nearing the end of the list.
-        guard indexPath.row >= self.conversationController.messages.count - 10 else { return }
-
-        self.conversationController.loadPreviousMessages()
-    }
-}
+        // Start fetching new messages once the user is nearing the end of the list.
+        guard indexPath.row < 10 else { return }
```

Comment:
Reversing this logic now that the messages are loaded in right to left.

Reply:
Fetching/Pagination will be intentional. No auto updating. I can explain.

Reply:
Make total sense to me, I'll be adding that feature in a later PR.